src/upload/system/library/transbank/utils/LogHandler.php

```diff
@@ -0,0 +1,139 @@
+<?php
+namespace Transbank\Opencart\Webpay;
+use Monolog\Logger;
+use Monolog\Handler\RotatingFileHandler;
+
+class LogHandler
+{
+
+    private $logDir;
+    private $logURL;
+    private $logger;
+
+    public function __construct($ecommerce = 'opencart')
+    {
+        $this->logDir = DIR_STORAGE."logs/Transbank_webpay";
+        $this->logURL = str_replace($_SERVER['DOCUMENT_ROOT'], "", $this->logDir);
+        $this->logger = new Logger('webpay_logger');
+        $this->logger->pushHandler(new RotatingFileHandler("{$this->logDir}/log_transbank_{$ecommerce}.log", 10, Logger::DEBUG));
+    }
+
+    /**
+     * Formats the size of a file in readable format.
+     * @param string $path
+     * @return string
+     */
+    private function formatBytes($path): string
+    {
+        $bytes = @filesize($path);
+        if ($bytes > 0) {
+            $unit = intval(log($bytes, 1024));
+            $units = array('B', 'KB', 'MB', 'GB');
+            if (array_key_exists($unit, $units) === true) {
+                return sprintf('%d %s', $bytes / pow(1024, $unit), $units[$unit]);
+            }
+        }
+        return '0 B';
+    }
+
+    /**
+     * Returns the directory where the logs are stored.
+     * @return string
+     */
+    private function getLogDir(): string
+    {
+        return $this->logDir;
+    }
+
+    /**
+     * Generates a list of log files.
+     * @return array|null
+     */
+    private function setLogList(): array
```

Comment:
This declares that it returns an array, but it can also return `null`

Reply:
The doc for the method is wrong, the method is a "`set`", but the description indicates that it generates a list

Reply:
The name of the function will be changed to a more descriptive one, the documentation will be updated and the code will be modified.